### ISSUE TITLE
Add to shelf button

### ISSUE THREAD

**Issue by wisbit** [leram84/layer.Cake#18]
Hello,
First and foremost..
AWESOME !! your theme looks very very good! Appreciate all the work.
I just want to report something that might be my mistake, but here goes, there doesn't seem to be an "add to shelf" button after doing a simple search, which there is in the standard theme.
Thanks for the feedback

**Comment by mmol67** [comment]
Yes, please. I like this theme a lot, but I miss the "add to shelf" button too.

**Comment by dr400** [comment]
I can but just agree
